README.md

```diff
@@ -3,6 +3,16 @@ Integration between Terraform and Docker.
 
 This project is intended for people who want to build their docker images from terraform.
 
+## Some advise
+Now the latest version images will come by default with Terraform v0.13.0 or higher for the correct operation of the command `terraform login`.
+
+Some previous versions of the images remain available on Docker Hub with their respective latest tags.
```

Comment:
with their respective tags *.

latest is misleading.

Reply:
Fixed